app/scripts/modules/core/src/managed/constraints/ConstraintCard.less

```diff
@@ -0,0 +1,51 @@
+.ConstraintCard {
+  // TODO (erik): this is a bunch of custom styles for a button.
+  // really it should be a real button component we put somewhere central.
+  // for now it's entirely experimental and might totally go away
+  // or be replaced.
+
+  .constraint-override-action {
```

Comment:
pay no attention to this. I needed to invent a new-looking button but haven't actually built it out properly yet.